### ISSUE TITLE
k8s-ci-robot automatically approve PRs

### ISSUE THREAD

**Issue by IronPan** [kubernetes/test-infra#10721]
**What happened**:
The https://github.com/k8s-ci-robot starting to automatically approve PRs as soon as they are created

**What you expected to happen**:
Not approve untill add /approve comment 

**How to reproduce it (as minimally and precisely as possible)**:
Create a PR in https://github.com/kubeflow/pipelines

**Please provide links to example occurrences, if any**:
https://github.com/kubeflow/pipelines/pull/672
https://github.com/kubeflow/pipelines/pull/671
https://github.com/kubeflow/pipelines/pull/670

**Anything else we need to know?**:

**Comment by krzyzacy** [comment]
is there a recent change? @fejta @cjwagner 

https://github.com/kubeflow/pipelines/blob/master/frontend/OWNERS#L2 @rileyjbauer is an owner for all the file in the PR so the PR will be approved by default, you can turn off implicit self approval if you don't want that?

**Comment by IronPan** [comment]
Is this a recent change that owner's change will granted with /approve by default? I don't think that's what was happening before.

**Comment by IronPan** [comment]
could you forward me instruction how to turn implicit self approval off? thanks!

**Comment by jlewi** [comment]
I think this might be an improvement.

IIUC it used to be the case that if someone in the OWNERs file sends out a PR for approval; if that PR is LGTM'd by someone not in the approvers file. Then the PR author would have to do "/approve". 

So with the new behavior it seems like if PR author is also an approver they might nolonger need to explicitly add "/approve"; the LGTM is sufficient

If a PR author doesn't want a PR to be merged automatically; then they could mark it WIP or explicitly place a hold.

**Comment by cjwagner** [comment]
Yes, the default behavior was recently changed: https://github.com/kubernetes/test-infra/pull/10435 @fejta

**Comment by Ark-kun** [comment]
Now it's automatically LGTMing PRs: https://github.com/kubeflow/pipelines/pull/662#event-2069939092

/reopen

**Comment by Ark-kun** [comment]
Sorry, I got confused because GitHub only updated the main discussion thread, but not the comment thread, so there wasn't any LGTM comment when I searched.
The robot started auto-approving PRs today so I thought it was the case here.

/close
